packages/donor/src/components/Wizard/WizardScreen.tsx

```diff
@@ -1,104 +1,88 @@
-import Button from "../basic/Button/Button";
-import Text from "../basic/Text";
+import Button, { ButtonVariant } from "../basic/Button";
 import styles from "./WizardScreen.module.scss";
 import { useState } from "react";
-import { makeStyles } from "@material-ui/core/styles";
 import MobileStepper from "@material-ui/core/MobileStepper";
-import { BrowserRouter, Link } from "react-router-dom";
 import { PropTypes } from "@material-ui/core";
-import { ButtonVariant } from "../basic/Button";
-
-const useStyles = makeStyles({
-  root: {
-    flexDirection: "column",
-    backgroundColor: "#fff",
-  },
-
-  dot: {
-    margin: "8px",
-  },
-
-  dots: {
-    margin: "30px 0",
-  },
-
-  dotActive: {
-    backgroundColor: "#A2D45E",
-  },
-});
+import ZMScreen from "../basic/ZMScreen";
 
 interface WizardPage {
   imageUrl: string;
   title: string;
-  content: { text: string; bold?: boolean; link?: string; linkText?: string }[];
+  content: {
+    text: string;
+    bold?: boolean;
+    button?: { text: string; onClick: () => void };
+  }[];
   buttonText: string;
   buttonVariant: ButtonVariant;
   buttonColor: PropTypes.Color;
 }
+
 export interface WizardScreenProps {
   pages: WizardPage[];
-  linkRow?: { text: string; linkText: string; link: string };
   onFinish: () => void;
 }
 
 export default function WizardScreen(props: WizardScreenProps) {
-  const classes = useStyles();
   const [currentStep, setCurrentStep] = useState(0);
   const onNext = () => {
     if (currentStep < props.pages.length - 1) setCurrentStep(currentStep + 1);
     else props.onFinish();
   };
   return (
-    <div className={styles.container}>
+    <ZMScreen className={styles.container}>
       <img
         className={styles.image}
         src={props.pages[currentStep].imageUrl}
         alt=""
       />
-      <Text className={styles.title}>{props.pages[currentStep].title}</Text>
+      <div className={styles.title}>{props.pages[currentStep].title}</div>
       <div className={styles.content}>
         {props.pages[currentStep].content.map((key, index) => (
           <div>
-            <Text
+            <div
               key={index}
-              className={key.link ? styles.textLink : ""}
+              className={key.button ? styles.buttonText : ""}
               style={
                 key.bold ? { fontWeight: "bold" } : { fontWeight: "normal" }
               }
             >
               {key.text}
-              {key.link ? (
-                <BrowserRouter>
-                  <Link className={styles.link} to={key.link}>
-                    {key.linkText}
-                  </Link>
-                </BrowserRouter>
-              ) : null}
-            </Text>
+
+              {key.button && (
+                <Button
+                  title={key.button.text}
+                  onClick={key.button.onClick!}
+                  variant={ButtonVariant.text}
```

Comment:
this button font is smaller then the rest of the content, it should be the same size